test/metabase/api/collection_test.clj

```diff
@@ -914,7 +984,6 @@
       (is (partial= lucky-personal-subcollection-item
                     (api-get-lucky-personal-collection-with-subcollection :crowberto))))))
 
-
```

Comment:
nit: We often do have two newlines _before_ a new section of code. Not sure if this is a real convention or not though tbh